# Exploración inicial: delitos de hurto por modalidades

**Archivo:** `Reporte_Hurto_por_Modalidades_Policía_Nacional_20260916.csv`

**Pregunta inicial:** 
¿Es posible predecir la cantidad de hurtos de motocicletas que se presentarán en un municipio durante el siguiente mes, utilizando el comportamiento histórico de los hurtos y las variables disponibles en el dataset?

In [9]:
import pandas as pd

pd = pd.read_csv(
    "../datos/Reporte_Hurto_por_Modalidades_Policía_Nacional_20260916.csv"
    )

pd.head()


,DEPARTAMENTO,MUNICIPIO,CODIGO DANE,ARMAS MEDIOS,FECHA HECHO,GENERO,GRUPO ETARIO,TIPO DE HURTO,CANTIDAD
0,ATLÁNTICO,Barranquilla (CT),8001000,NO REPORTADO,05/03/2014,FEMENINO,ADULTOS,ARTÍCULO 239. HURTO AUTOMOTORES,1
1,META,Villavicencio (CT),50001000,NO REPORTADO,14/02/2014,FEMENINO,ADULTOS,ARTÍCULO 239. HURTO AUTOMOTORES,1
2,NORTE DE SANTANDER,Los Patios,54405000,NO REPORTADO,21/02/2014,MASCULINO,ADULTOS,ARTÍCULO 239. HURTO MOTOCICLETAS,1
3,ANTIOQUIA,Amagá,5030000,NO REPORTADO,03/01/2014,MASCULINO,ADULTOS,ARTÍCULO 239. HURTO MOTOCICLETAS,1
4,META,Villavicencio (CT),50001000,NO REPORTADO,22/02/2014,MASCULINO,ADULTOS,ARTÍCULO 239. HURTO AUTOMOTORES,1


In [10]:
pd.info()

<class 'pandas.DataFrame'>
RangeIndex: 656635 entries, 0 to 656634
Data columns (total 9 columns):
 #   Column         Non-Null Count   Dtype
---  ------         --------------   -----
 0   DEPARTAMENTO   656635 non-null  str  
 1   MUNICIPIO      656635 non-null  str  
 2   CODIGO DANE    656635 non-null  int64
 3   ARMAS MEDIOS   656635 non-null  str  
 4   FECHA HECHO    656635 non-null  str  
 5   GENERO         655923 non-null  str  
 6   GRUPO ETARIO   656430 non-null  str  
 7   TIPO DE HURTO  656635 non-null  str  
 8   CANTIDAD       656635 non-null  int64
dtypes: int64(2), str(7)
memory usage: 103.6 MB


In [11]:
print(f'Filas: {pd.shape[0]:,}')
print(f'Columnas: {pd.shape[1]}')

print('Nombres de las columnas:')
for numero, columna in enumerate(pd.columns, start=1):
    print(f'{numero}. {columna}')

Filas: 656,635
Columnas: 9
Nombres de las columnas:
1. DEPARTAMENTO
2. MUNICIPIO
3. CODIGO DANE
4. ARMAS MEDIOS
5. FECHA HECHO
6. GENERO
7. GRUPO ETARIO
8. TIPO DE HURTO
9. CANTIDAD


In [12]:
pd.isnull().sum()

DEPARTAMENTO       0
MUNICIPIO          0
CODIGO DANE        0
ARMAS MEDIOS       0
FECHA HECHO        0
GENERO           712
GRUPO ETARIO     205
TIPO DE HURTO      0
CANTIDAD           0
dtype: int64

## Resultado inicial

- El dataset contiene información histórica sobre casos de hurto registrados por la Policía Nacional, reportando 656,635 datos.
- Los registros contienen información geográfica, temporal y características relacionadas con los hechos.
- La variable de fecha permitirá organizar los registros cronológicamente.
